backend/ml/management/commands/ml_train.py

```diff
@@ -130,17 +159,36 @@ def save_data(video_scores, contributor_rating_scores):
         ]
     )
 
+def process(trusted_only=True):
+    criterias_list = Poll.default_poll().criterias_list
+    comparison_data = fetch_data(trusted_only=trusted_only)
+    glob_score, loc_score = ml_run(
+        comparison_data, criterias=criterias_list, save=True, verb=-1
+    )
+    save_data(glob_score, loc_score, trusted_only=trusted_only)
+
+
 
 class Command(BaseCommand):
     help = "Runs the ml"
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--skip_untrusted',
+            nargs='?',
+            help='Skip ML run on untrusted users',
+        )
```

Comment:
This optional argument is only added for test purpose. Do you think it's relevent @amatissart ?